socorro/processor/general_transform_rules.py

```diff
@@ -16,25 +16,22 @@ def action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
 
 class CPUInfoRule(Rule):
     def action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
-        cpu_name = ''
-        cpu_info = ''
-
-        system_info = processed_crash.get('json_dump', {}).get('system_info')
-        if system_info:
-            cpu_name = system_info.get('cpu_arch', '')
-
-            if 'cpu_info' in system_info and 'cpu_count' in system_info:
-                cpu_info = (
-                    '%s | %s' % (
-                        system_info['cpu_info'],
-                        system_info['cpu_count']
-                    )
-                )
-            else:
-                cpu_info = system_info.get('cpu_info', '')
-
-        processed_crash['cpu_name'] = cpu_name
-        processed_crash['cpu_info'] = cpu_info
+        # This is the CPU that the product was built for
+        processed_crash['cpu_arch'] = glom(
+            processed_crash, 'json_dump.system_info.cpu_arch', default=''
+        )
+        # NOTE(willkg): "cpu_name" is deprecated and we can remove it in July 2019
+        processed_crash['cpu_name'] = glom(
+            processed_crash, 'json_dump.system_info.cpu_arch', default=''
+        )
```

Comment:
Because we're still using it for Elasticsearch and the API, we need to keep `cpu_name` until then.